.gitignore

```diff
@@ -105,3 +105,4 @@ venv.bak/
 
 # IntelliJ
 .idea
+Pipfile*
```

Comment:
@avallbona could you add a line at the end of the file?

```suggestion
Pipfile*

```

Reply:
done